scripts/build.sh

```diff
@@ -1,40 +1,65 @@
 #!/usr/bin/env bash
 
-set -e
+set -xe
 
 echo "***** Start: Building minirhumba installer *****"
 
-CONSTRUCT_ROOT="${CONSTRUCT_ROOT:-/construct}"
+CONSTRUCT_ROOT="${CONSTRUCT_ROOT:-$PWD}"
 
 cd $CONSTRUCT_ROOT
 
+# Constructor should be latest for non-native building
+# See https://github.com/conda/constructor
 echo "***** Install constructor *****"
-conda install -y -c conda-forge conda-standalone jinja2 R
-python -m pip install https://github.com/conda/constructor/archive/master.zip
-conda list
+conda install mamba -c conda-forge
```

Comment:
if you do this you can just use mambaforge.

Reply:
I guess :)